gnosis/safe-react
# feat: track create/load Safe

## What it solves
Tracking of Safe creation/loading

## How this PR fixes it
The `Track` component has been extended to include a `desc` prop and the following parameters are now tracked

- Back navigation inside `Stepper` (on first step 'Cancel' `desc` and after 'Back', with `{ step }` `payload`).
- Next navigation inside `Stepper` (on last step 'Finish' `desc` and before 'Next', with `{ step }` `payload`).
- A one-time Safe-name flag in the creation/loading stepper ('Set name' `desc`).
- Creation/loading review steps submission ('Create'/'Load' `desc`, with `{ owners, threshold}` `payload`.

## How to test it
Navigate through 'Create'/'Load' Safe stepper and observe tracking in the console from GTM.
